src/pages/addCard/components/AddCardForm.tsx

```diff
@@ -0,0 +1,50 @@
+import { FormEvent, useRef, useState } from "react";
+import CardNumberInput from "./CardNumberInput";
+import CardExpiredDateInput from "./CardExpiredDateInput";
+import CardSecurityCodeInput from "./CardSecurityCodeInput";
+import CardHolderNameInput from "./CardHolderNameInput";
+import CardPasswordInput from "./CardPasswordInput";
+
+export default function AddCardForm() {
+  const [cardNumber, setCardNumber] = useState<string>("");
+  const [cardExpiredDate, setCardExpiredDate] = useState<string>("");
+  const [cardHolderName, setCardHolderName] = useState<string>("");
+  const cardSecurityCodeRef = useRef<HTMLInputElement>(null);
+  const firstLetterOfPasswordRef = useRef<HTMLInputElement>(null);
+  const secondLetterOfPasswordRef = useRef<HTMLInputElement>(null);
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log("cardNumber", cardNumber);
+    console.log("cardExpiredDate", cardExpiredDate);
+    console.log("cardHolderName", cardHolderName);
+    console.log("securityCodeRef", cardSecurityCodeRef.current?.value);
+    console.log(
+      "firstLetterOfPasswordRef",
+      firstLetterOfPasswordRef.current?.value
+    );
+    console.log(
+      "secondLetterOfPasswordRef",
+      secondLetterOfPasswordRef.current?.value
+    );
+  };
+  return (
+    <form onSubmit={handleSubmit}>
+      <CardNumberInput cardNumber={cardNumber} setCardNumber={setCardNumber} />
+      <CardExpiredDateInput
+        cardExpiredDate={cardExpiredDate}
+        setCardExpiredDate={setCardExpiredDate}
+      />
+      <CardHolderNameInput
+        cardHolderName={cardHolderName}
+        setCardHolderName={setCardHolderName}
+      />
+      <CardSecurityCodeInput cardSecurityCodeRef={cardSecurityCodeRef} />
```

Comment:
```suggestion
      <CardSecurityCodeInput ref={cardSecurityCodeRef} />
```
이렇게 표현할 수 있도록 forwardRef로 컴포넌트를 만들어주면 어떨까요?

Reply:
제안해주신 코드가 더 깔끔한 것 같아요! 참고하겠습니다.